concourse/enumerator.py

```diff
@@ -192,9 +208,14 @@ def _branch_cfg_or_none(
                 path='branch.cfg',
                 ref='refs/meta/ci',
             ).decoded.decode('utf-8')
-            return BranchCfg(raw_dict=load_yaml(branch_cfg))
+            info(f'Linting branch cfg for {repository}')
+            lint_yaml(branch_cfg, config=REPLICATION_LINTING_CFG)
         except NotFoundError:
             return None # no branch cfg present
+        except LintingError as e:
```

Comment:
I agree with introducing a new exception. However: why not fail?

Reply:
mh?

Reply:
originally, the Idea was to avoid failing our central replication. I've moved the exception handling so that this is done in a more elegant fashion